tests/integration/test_render.py

```diff
@@ -32,18 +33,22 @@ class TestDragAndDropRender(BaseIntegrationTest):
     PAGE_TITLE = 'Drag and Drop v2'
     PAGE_ID = 'drag_and_drop_v2'
     ITEM_PROPERTIES = [{'text': '1'}, {'text': '2'}, {'text': 'X'}, ]
+    SIDES = ['Top', 'Bottom', 'Left', 'Right']
 
-    def load_scenario(self, item_background_color="", item_text_color=""):
+    def load_scenario(self, item_background_color="", item_text_color="", zone_labels=False, zone_borders=False):
+        json = load_resource("integration/data/test_data_a11y.json")
```

Comment:
@agaylard It would be more accurate to name this variable `data` (and to also use that name in the `scenario_xml` template below): DnDv2 blocks have a `data` field, and this line loads the value for this field. (Also, `load_resource` calls `pkg_resources.resource_string`, which I think is agnostic to the format of the data that it loads).